xml/System.Threading/Timer.xml

```diff
@@ -53,12 +53,12 @@ Sub TimerCallback(state As Object)
  The timer delegate is specified when the timer is constructed, and cannot be changed. The method does not execute on the thread that created the timer; it executes on a <xref:System.Threading.ThreadPool> thread supplied by the system.  
   
 > [!TIP]
->  The .NET Framework Class Library includes four classes named `Timer`, each of which offers different functionality:  
->   
-> -   <xref:System.Timers.Timer?displayProperty=nameWithType>, which fires an event and executes the code in one or more event sinks at regular intervals. The class is intended for use as a server-based or service component in a multithreaded environment; it has no user interface and is not visible at runtime.  
-> -   <xref:System.Threading.Timer?displayProperty=nameWithType>, which executes a single callback method on a thread pool thread at regular intervals. The callback method is defined when the timer is instantiated and cannot be changed. Like the <xref:System.Timers.Timer?displayProperty=nameWithType> class, this class is intended for use as a server-based or service component in a multithreaded environment; it has no user interface and is not visible at runtime.  
-> -   <xref:System.Windows.Forms.Timer?displayProperty=nameWithType>, a Windows Forms component that fires an event and executes the code in one or more event sinks at regular intervals. The component has no user interface and is designed for use in a single-threaded environment; it executes on the UI thread.  
-> -   <xref:System.Web.UI.Timer?displayProperty=nameWithType>, an ASP.NET component that performs asynchronous or synchronous web page postbacks at a regular interval.  
+> .NET includes four classes named `Timer`, each of which offers different functionality:  
```

Comment:
This can't be simply changed to .NET from .NET Framework without some clarification of the specific APIs. Two of the APIs, System.Windows.Forms.Timer and System.Web.UI.Timer, are available in the .NET Framework only and are not available in either .NET Core or .NET Standard.
@